.docker/php-dev.ini

```diff
@@ -1 +1,4 @@
 opcache.validate_timestamps = 1
+
+xdebug.remote_enable = 1
+xdebug.remote_host = host.docker.internal
```

Comment:
This is the host where an IDE runs, I think. Might be worth making `xdebug.remote_port` explicit rather than relying on the default.